draft-xyz-rats-psa-endorsements.md

```diff
@@ -131,94 +140,149 @@ firmware measurement associated with Implementation ID
+before provisioning the endorsements.
+
+### Atomic
+{: #sec-atmoic}
+In an atomic provisioning model, measurements for each firmware component associated with a hardware RoT is conveyed using a unique CoMID tag. Each tag contains the identifier for the platform RoT associated to the PSA device.
+This enables the verifier to link all the firmware components belonging to the same platform and model them as a collection of nodes in a graph.
+
+Each node in the graph is a CoMID, which has a Reference Value Map that contains measurements associated with a specific version of firmware component. Each firmware component can have its own upgrade chain which can be tracked. Each tagged edge is the CoMID-to-CoMID link relation describing an update or patch action on the target node by the source node.
 
 ~~~ goat
-{::include art/one-comid-per-refval-updates.txt}
+{::include art/atomic.txt}
 ~~~
-{: #fig-device-refval-updates title="Firmware update graph"}
+{: #fig-atomic title="Atomic Provisioning Model"}
 
-### CoMID Layout
+A limitation of this model is that it may not be able to distinguish between a valid software RoT ( i.e. an
+expected combination of software component versions) from an invalid one.
 
-Depending on the firmware update strategy, Endorsers have the following
-two packing options:
+### Bundled
+{: #sec-bundled}
+In the bundled provisioning model all firmware components associated with a hardware RoT are grouped together in a single
+CoMID tag. The CoMID tag contains the identifier for the platform RoT associated to the PSA device. CoMID tag has a Reference Value Map which is used to carry a list of Reference Values ({{sec-ref-values}}). Each entry in the list carries the measurements associated to a firmware component. This model is applicable if the device firmware is made of separately measurable components however these components are always installed together in a single firmware update operation. In this model, the evolution of the versions of firmware component is collectively tracked using the CoMID updates and patches relations.
 
-1. Put multiple Reference Values in one CoMID;
-2. Use one CoMID for each Reference Value.
+~~~ goat
+{::include art/bundled.txt}
+~~~
+{: #fig-bundled title="Bundled Provisioning Model"}
 
-The first packing option MAY be used if the device firmware is made of
-separately measurable components and these components are always installed
-together in a single software update operation.  If differential software
-updates are possible (or if there is only one measurable firmware component),
-Endorsers MUST use the second packing option.
+Each tagged edge is the CoMID-to-CoMID link relation describing an
+update or patch action on the target node by the source node.
```

Comment:
content is OK but it needs some reorg / reflow.

Reply:
have used bullets now for Atomic and Bundled and added links to the individual sections, so that it connects nicely!